### PR TITLE
fix #109136, fix #33201: more articulations in palette

### LINKED ISSUE #33201: Де в новій версії MuseScore є розділ "інструменти", коли вже створену партитуру і хочеш прибрати певні інструменти. І де є розділ партії щоб зробити партії?
Де в новій версії MuseScore є розділ "інструменти", коли вже створену партитуру і хочеш прибрати певні інструменти. І де є розділ партії щоб зробити партії?

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Missing: support for these in Emmentaler and Gonville, as well as proper vtests

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
Need to check http://www.smufl.org/version/latest/range/articulation/ whether to add more and which.

**Comment by Jojo-Schmitz** [comment]
Checking http://www.smufl.org/version/latest/range/articulation/ whether to add more and which reveals the following to be missing:  StaccatissimoWedge, StaccatissimoStroke, TenutoAccent, Stress, Unstress, LaissezVibrer, MarcatoTenuto

**Comment by Jojo-Schmitz** [comment]
@mgaviolo: while looking into adding glyphs to Emmentaler, maybe you can also handle the ones from https://musescore.org/en/node/125436

**Comment by Jojo-Schmitz** [comment]
Very strange, on Travis it doesn't abend

**Comment by Jojo-Schmitz** [comment]
found the cause for the crash: when adding articulations we also need to add styles für them

File: mscore/exportxml.cpp [hunk 1]
```diff
@@ -2150,6 +2181,17 @@ void ExportMusicXml::chordAttributes(Chord* chord, Notations& notations, Technic
...
+                  case ArticulationType::Harmonic:
+#if 0 // TODO
+                        {
+                        notations.tag(xml);
+                        technical.tag(xml);
+                        xml.stag("harmonic");
+                        xml.tagE("natural");
+                        xml.etag();
+                        }
+#endif
```
Comment (Jojo-Schmitz) [inline mscore/exportxml.cpp]: Here I'd like @lvinken to case an eye

File: mscore/importmxmlpass2.cpp [hunk 2]
```diff
@@ -1230,6 +1230,9 @@ static bool addMxmlArticulationToChord(ChordRest* cr, QString mxmlName)
       map["turn"]             = ArticulationType::Turn;
       map["inverted-turn"]    = ArticulationType::Reverseturn;
       map["stopped"]          = ArticulationType::Plusstop;
+#if 0 //TODO
+      map["harmonic"]         = ArticulationType::Harmonic;
+#endif
```
Comment (Jojo-Schmitz) [inline mscore/importmxmlpass2.cpp]: Here too I'd like @lvinken to cast an eye

File: mscore/exportxml.cpp [hunk 3]
```diff
@@ -2150,6 +2181,17 @@ void ExportMusicXml::chordAttributes(Chord* chord, Notations& notations, Technic
...
+                  case ArticulationType::Harmonic:
+/* TODO
+                        {
+                        notations.tag(xml);
+                        technical.tag(xml);
+                        xml.stag("harmonic");
+                        xml.tagE("natural");
+                        xml.etag();
+                        }
+*/
```
Comment (Jojo-Schmitz) [inline mscore/exportxml.cpp]: I believe this should be correct, but I'd rather like @lvinken to cast an eye on this

**Comment by Jojo-Schmitz** [comment]
OK, rebase needed...

**Comment by Jojo-Schmitz** [comment]
rebased, next step is to get them translated too

**Comment by Jojo-Schmitz** [comment]
And, at least for the Palette, to get rid of the " above" part of some of the articulations (e.g. "Fermata above"

**Comment by Jojo-Schmitz** [comment]
Oh Boy, I hope a rebase won't be too difficult.

**Comment by Jojo-Schmitz** [comment]
I'm affraid it is broken beyond repair. Will leave this PR open, while preparing a new one.

**Comment by Jojo-Schmitz** [comment]
see #2854

File: mscore/exportxml.cpp (hunk 3 above)
Comment (lvinken) [inline mscore/exportxml.cpp]: Hi Jojo, have been too busy to reply in time. Let me know if there is anything I can do to help.

**Pull request closed without merging**